python-connectors/googlesheets-sheet/connector.json

```diff
@@ -9,12 +9,34 @@
     "canCountRecords" : true,
 
     "params": [
+        {
+            "name": "auth_type",
+            "label": "Type of authentication",
+            "type": "SELECT",
+            "selectChoices": [
+                {
+                    "value": "service-account",
+                    "label": "Service account preset"
+                },
+                {
+                    "value": "credentials",
+                    "label": "Service account credentials (legacy)"
+                }
+            ]
```

Comment:
Same points as in recipe json re naming

Reply:
👍 Done in [0ae3fbb](https://github.com/dataiku/dss-plugin-googlesheets/pull/2/commits/0ae3fbb57d6aee92b5fdfdcf37504229875a8a26)